cfme/common/physical_server_views.py

```diff
@@ -0,0 +1,188 @@
+        _dropdown = Dropdown(text=Parameter("button_group"))
+
+        def item_select(self, button, handle_alert=False):
+            self._dropdown.item_select(button, handle_alert=handle_alert)
+
+
+class PhysicalServerDetailsEntities(View):
+    """Represents Details page."""
+    properties = SummaryTable(title="Properties")
+    networks = SummaryTable(title="Networks")
+    relationships = SummaryTable(title="Relationships")
+    power_management = SummaryTable(title="Power Management")
+    assets = SummaryTable(title="Assets")
+    firmware = SummaryTable(title="Firmware")
+    network_devices = SummaryTable(title="Network Devices")
+    smart = SummaryTable(title="Smart Management")
+
+
+class PhysicalServerDetailsView(ComputePhysicalInfrastructureServersView):
+    """Main PhysicalServer details page."""
+    breadcrumb = BreadCrumb(locator='.//ol[@class="breadcrumb"]')
+    toolbar = View.nested(PhysicalServerDetailsToolbar)
+    entities = View.nested(PhysicalServerDetailsEntities)
+
+    @property
+    def is_displayed(self):
+        title = "{name} (Summary)".format(name=self.context["object"].name)
+        return (self.in_compute_physical_infrastructure_servers and
+                self.breadcrumb.active_location == title)
+
+
+class PhysicalServerTimelinesView(TimelinesView, ComputePhysicalInfrastructureServersView):
+    """Represents a PhysicalServer Timelines page."""
+
+    @property
+    def is_displayed(self):
+        return (self.in_compute_physical_infrastructure_servers and
+                super(TimelinesView, self).is_displayed)
+
+
+class PhysicalServerManagePoliciesView(BaseLoggedInPage):
+    """PhysicalServer's Manage Policies view."""
+    policies = BootstrapTreeview("protectbox")
+    entities = View.nested(BaseNonInteractiveEntitiesView)
+    save_button = Button("Save")
+    reset_button = Button("Reset")
+    cancel_button = Button("Cancel")
+
+    @property
+    def is_displayed(self):
+        return False
+
+
+class PhysicalServersToolbar(View):
+    """Represents hosts toolbar and its controls."""
+    configuration = Dropdown(text="Configuration")
+    policy = Dropdown(text="Policy")
+    lifecycle = Dropdown(text="Lifecycle")
+    monitoring = Dropdown(text="Monitoring")
+    power = Dropdown(text="Power")
+    identify = Dropdown(text="Identify")
+    view_selector = View.nested(ItemsToolBarViewSelector)
+
+
+class PhysicalServerSideBar(View):
+    """Represents left side bar. It usually contains navigation, filters, etc."""
+    pass
+
+
+class PhysicalServerEntitiesView(BaseEntitiesView):
+    """Represents the view with different items like hosts."""
+    @property
+    def entity_class(self):
+        return PhysicalServerEntity().pick(self.browser.product_version)
+
+
+class PhysicalServersView(ComputePhysicalInfrastructureServersView):
+    toolbar = View.nested(PhysicalServersToolbar)
+    sidebar = View.nested(PhysicalServerSideBar)
+    paginator = PaginationPane()
```

Comment:
So the views here are highly nested and conditional when using BaseEntitiesView, and it actually already includes the widget definitions for paginator and title - so you can drop this widget definition, it just has to be accessed through `view.entities.paginator`.

https://github.com/ManageIQ/integration_tests/blob/4addd2881a759c29abcab6f1d0f799d9f56bc94e/widgetastic_manageiq/__init__.py#L2899